packages/network/src/network.ts

```diff
@@ -11,7 +11,7 @@ export interface NetworkConfig {
 }
 
 export const StacksNetworks = ['mainnet', 'testnet', 'devnet', 'mocknet'] as const;
-export type StacksNetworkName = (typeof StacksNetworks)[number];
+export type StacksNetworkName = typeof StacksNetworks[number];
```

Comment:
```suggestion
export type StacksNetworkName = (typeof StacksNetworks)[number];
```

Can we revert this for the linter?